tests/infra/proposal.py

```diff
@@ -18,11 +18,11 @@ def __init__(self, proposal):
 
 # Values defined in node/proposals.h
 class ProposalState(Enum):
-    Open = "OPEN"
-    Accepted = "ACCEPTED"
-    Withdrawn = "WITHDRAWN"
-    Rejected = "REJECTED"
-    Failed = "FAILED"
+    OPEN = "Open"
+    ACCEPTED = "Accepted"
+    WITHDRAWN = "Withdrawn"
+    REJECTED = "Rejected"
+    FAILED = "Failed"
```

Comment:
None of the other Python enum names have changed, only this one. If we're not touching the code names, only the values, the left hands here should be reverted.

Reply:
Hm, I didn't want to change C++ enums but Python test code seems ok? Is there anything that could break?

Reply:
Its harmless, just seems odd to do a partial consistency fix so I assume it was a mistake.

Reply:
I fixed the ones that are in sync with C++. There are some local ones which I didn't touch.